\begin{rubric}{Internships}
%
% Blank lines result in extra space!
%
\subrubric{During M.Sc.}
\entry*[Jan -- Jun 2014]%
	\textbf{Connecting multiple gene expression signatures with candidate drugs for boosting heart regeneration potential.} \par
	\textbf{Supervisor} Dr. Francisco Azuaje \par
	\textbf{Institute} CRD-Sante\'e, Luxembourg, Luxembourg \par
	\textbf{Grade} 9/10 \par
	\url{https://github.com/npklein/drugFinder}
%Identified novel drug candidates which can boost the heart regeneration of zebrafish using microarray expression data.
%
\entry*[Aug -- Dec 2013]%
\textbf{Clustering of MS/MS spectra to improve peptide and protein identification.} \par
\textbf{Supervisor} Dr. Tham Pham \par
\textbf{Institute} Cancer Center Amsterdam, Amsterdam, Netherlands\par
\textbf{Grade} 9/10 \par
%Clustered unidentified spectra from mass spectrometer measurements together with identified spectra to improve peptide and protein identification.
\subrubric{During B.Sc.}
\entry*[Feb -- Jun 2012]%
\textbf{Methods to assess the reproducibility and coverage of tandem mass spectrometry data in phosphoproteomics experiments.} \par
\textbf{Supervisor} Dr. David Martin \par
\textbf{Institute} Wellcome Trust, Dundee, Scotland \par
\textbf{Grade} 9/10 \par
%Developed a Python module to perform statistical analysis on mass spectrometry data.\par
\url{https://github.com/npklein/pyMSA}
%
\entry*[Sep -- Dec 2010]%
\textbf{How widespread are microProteins and how did they evolve?} \par
\textbf{Supervisor} Dr. Sue Rhee \par
\textbf{Institute} Carnegie Institue for Life Sciences, Stanford, USA \par
\textbf{Grade} 9/10 \par
%Wrote a program in Python to identify putative microProteins. \par
\url{https://github.com/npklein/miP3}
%
\end{rubric}